src/EventStore.Core/Services/Transport/Grpc/Streams.Read.cs

```diff
@@ -38,6 +38,11 @@ internal partial class Streams<TStreamId> {
 				var user = context.GetHttpContext().User;
 				var requiresLeader = GetRequiresLeader(context.RequestHeaders);
 
+				var uuidOption = options.UuidOption;
```

Comment:
You could use `uuidOption` on [line 77](https://github.com/EventStore/EventStore/pull/4158/files#diff-b4a144b42979d2bcf4d8d82ddb732874a39d13b3966f28fe363540f0eadf9893R77) instead of the call to `options.UuidOption`

Reply:
Done